elements/pfe-accordion/BaseAccordionHeader.ts

```diff
@@ -36,140 +34,102 @@ export abstract class BaseAccordionHeader extends LitElement {
 
   static override readonly shadowRootOptions = { ...LitElement.shadowRootOptions, delegatesFocus: true };
 
-  @property({ attribute: 'aria-controls', reflect: true }) ariaControls?: string;
-
-  @property({ reflect: true }) bordered?: 'true'|'false';
-
-  @property({ reflect: true }) icon?: string;
-
-  @property({ reflect: true, attribute: 'icon-set' }) iconSet?: string;
-
-  @observed
   @property({ type: Boolean, reflect: true }) expanded = false;
 
   @property({ reflect: true, attribute: 'heading-text' }) headingText = '';
 
   @property({ reflect: true, attribute: 'heading-tag' }) headingTag = 'h3';
 
-  /**
-   * Sets color theme based on parent context
-   */
-  @colorContextConsumer()
-  @property({ reflect: true }) on?: ColorTheme;
-
-  @query('.toggle') button?: HTMLButtonElement;
-
-  private _generatedHtag?: HTMLHeadingElement;
+  #generatedHtag?: HTMLHeadingElement;
 
-  private slots = new SlotController(this, 'accents', null);
-
-  private logger = new Logger(this);
-
-  private get ariaExpandedState(): 'true'|'false' {
-    return String(!!this.expanded) as 'true'|'false';
-  }
-
-  constructor() {
-    super();
-    this.addEventListener('click', this._clickHandler);
-  }
+  #logger = new Logger(this);
 
   override connectedCallback() {
     super.connectedCallback();
+    this.addEventListener('click', this.#onClick);
     this.hidden = true;
     this.id ||= getRandomId('pfe-accordion');
```

Comment:
Probably want to update this to `getRandomId('accordion')` like `BaseAccordionPanel.ts`

Reply:
good call. will send a fix

Reply:
used the `localName`